src/builder.rs

```diff
@@ -240,7 +240,7 @@ impl DbcBuilder {
         if pk_set.len() != 1 {
             return Err(Error::ReissueSharePublicKeySetMismatch);
         }
-        let mint_public_key_set = match pk_set.iter().next() {
+        let mint_root_public_key_set = match pk_set.iter().next() {
```

Comment:
`mint_public_key_set` was a good name, lets not introduce a new "root" key terminology.

Reply:
I changed it back.